docs/blog/posts/community-dashboards.md

```diff
@@ -0,0 +1,154 @@
+func withResources(datasource string, clusterLabelMatcher *labels.Matcher) dashboard.Option {
+	labelMatchersToUse := []*labels.Matcher{
+		promql.JobVarV2,
+	}
+	labelMatchersToUse = append(labelMatchersToUse, clusterLabelMatcher)
+
+	return dashboard.AddPanelGroup("CPU Usage",
+		panelgroup.PanelsPerLine(1),
+		panelgroup.PanelHeight(10),
+		panelsGostats.CPUUsage(datasource, labelMatchersToUse...),
+	)
+}
+
+// Build the dashboard and add the time series panel withResources
+func BuildWorkloadOverview(project string, datasource string, clusterLabelName string) dashboards.DashboardResult {
+	clusterLabelMatcher := dashboards.GetClusterLabelMatcherV2(clusterLabelName)
+	return dashboards.NewDashboardResult(
+		dashboard.New("workload-overview",
+			dashboard.ProjectName(project),
+			dashboard.Name("Workload / Overview"),
+			dashboard.AddVariable("job",
+				listvariable.List(
+					labelvalues.PrometheusLabelValues("job",
+						labelvalues.Matchers("workload_build_info{}"),
+						dashboards.AddVariableDatasource(datasource),
+					),
+					listvariable.DisplayName("job"),
+				),
+			),
+			withResources(datasource, clusterLabelMatcher),
+		),
+	).Component("workload")
+}
+```
+
+## Beyond Copy-Pasting - Cleaner and Smarter way to building Dashboards
+
+Going back to the idea of copy-pasting raw YAML or JSON, although it might seem quicker, it leads to painful maintenance over time. The Go SDK offers a better path:
+
+* **Type safety** - You get compile-time feedback if something is wrong. No more mystery errors buried in JSON blobs.
+* **Reusability** - Define a panel once and reuse it wherever you want. Avoid duplication and drift.
+* **Maintainability** -  When a panel definition needs an update, you can do it in one place and apply it across all dashboards.
+* **CI/CD friendly** - Code is easier to version, review, and test. It fits naturally into modern GitOps workflows.
+
+Combining the use of panel libraries and the SDK, you get a flexible and scalable way to manage dashboards, while saving time, reducing errors, and staying consistent across your observability stack. This allows a community of dashboards users to have their lives made easier with these features. 
+
+Also considering the most commonly used dashboard across platforms and teams, like **Kubernetes**, **Prometheus** and exporters like **Node Exporter**, **Blackbox Exporter**, also projects like **Thanos** dashboards as it tends to be a power duo within the Prometheus ecosystem, we have from the get go some core dashboards that will help you in the long term in your observability journey.
+
+## Next Steps for the Community Dashboards
+
+We’re committed to expanding support for more CNCF projects, while continuing to enhance the existing panel libraries. By adding more features and improving the quality of reusable panels, we enable users to gain deeper insights from their dashboards—because the faster you can pinpoint an issue in your telemetry data, the better.
```

Comment:
```suggestion
We’re committed to expanding support for more CNCF and other OSS projects while continuing to enhance the existing panel libraries. By adding more features and improving the quality of reusable panels, we enable users to gain deeper insights from their dashboards because the faster you can pinpoint an issue in your telemetry data, the better.
```